web/src/components/tasks/business-formation/review/ReviewStep.test.tsx

```diff
@@ -773,4 +773,28 @@ describe("Formation - ReviewStep", () => {
       });
     });
   });
+
+  describe("when foreign corporation", () => {
```

Comment:
[dust] we could combine these to more concise

```
describe("when foreign", () => {
it.each(["c-corp, nonprofit"])("displays foreign cert of good standing when legalStructure is %s")....

it("displays will practice law when corp", async () = {})

it("doesn't display will practice law when nonprofit", async () = {})
})
```
the last test I'm assuming a typo exists below.

Reply:
Hm, I'd push back. I think this is a bit harder to read in my opinion. I think we tend to think of things on a legal-type by legal-type basis, ie, phrasing the tests as "here's what happens when foreign c-corp" and "here's what happens when foreign nonprofit" is more accurate to read from a documentation perspective than having to sort through all the tests in "when foreign" to identify which ones apply when. Like, if you're reading the tests trying to understand what's different for a foreign c-corp, it's easier to have that laid out together than to have to read all 3 individual `it` statements to identify which ones might have something to do with corp. 

Plus it would require a larger refactoring of the test file because there's a lot of other tests/describes for other "when foreign" cases that would require moving into this block.

Reply:
mhm, that makes sense, thank you! I thought a larger refactoring might be necessary too, left a comment above.